{{Toronto}}
<html>

<div>
<h5>Modelling Metabolic Activity with Flux-Balance Analysis</h5>
<p>We used Flux Balance Analysis (FBA) in order to model the metabolic activity of the E. coli S30 cell-free extract which has been used by our Wetlab team as the basis for the synthetic gene network driving the Pardee lab’s paper biosensor. We began by using an annotated, EcoCyc-aligned genome-scale reconstruction of the metabolic network SBML file for E. coli K12 MG1655 generated by Feist et al2., removed all transport, import, and export reactions from the metabolome, altered the reaction constraints to suit the conditions in LB media, and finally used COBRApy to model and optimize the metabolic activity of the E. coli cell extract.</p>
</p>
</div>

<div>
<h5>Data Mining Pipeline </h5>
<p>We constructed profile HMMs based on sets of sequences with functions related to metal binding and resistance to metal toxicity, where each profile HMM was constructed around one gene. After pulling annotated genomic information for all available bacterial species from EnsemblBacteria, we ran the nhmmer command in HMMER in order to locally align sections of genomic DNA with sections of the sequence sets making up our profile HMMs. Based on the nhmmer results, we created a table of annotations for all species which includes regions of match in genomic DNA (start and stop positions), annotations for all genes that received an alignment score above the default threshold, bit scores, E-values, and predicted bias. </p>
<p>We then stored annotated genomic sequence files (Genbank files), sequence positions (start and stop) nhmmer output, what operons each gene belongs to (using information from ODB3), KEGG Orthology annotations for operon function, phylogenetic profiles, and the profile HMMs discussed above in a postgreSQL relational database.  </p>
<p>Following this, we used the information in the database to train recurrent neural nets (RNNs) or MLPs to recognize operons. The first RNN determines whether or not any gene cluster entered as input is part of an operon, and the second RNN determines whether or not that operon has functions related to metal-binding. Thus, our pipeline allows us to identify the function of unknown operons. </p>
</div>

<div>
<h5>Smartphone Camera App for Colorimetric Analysis </h5>
<p>We used Apache Cordova to create a smartphone app for colorimetric analysis. The app was designed to analyze the output of the cell-free paper biosensor implemented for gold detection using the lacZ colour change by our Wetlab team. However, given that the app determines the base colour directly from the image, the app has wide-ranging capabilities that make it useful for analyzing reaction data from any one-to-one colour change. In response to a given trigger RNA, LacZ will cleave yellow chlorophenol red-b-D-galactopyranoside within the paper disc platforms, resulting in a purple chlorophenol red product. This colour change presents a colour intensity from which analyte concentration can be calculated. </p> 
<p>When opening the app, the user will be prompted to designate the following configurations: the aspect ratio of the biosensor paper, the (labelled) number of rows and columns within the wells, and the row-column coordinate for the well containing only yellow pigments. Our app’s image processing capabilities allow us to use information about the aspect ratio of the paper to construct a translucent frame in the camera’s live preview mode so that the user can more easily frame the paper, and the app includes an image colour summarizer that gets the image, converts it to LCH colourspace, and shows us the colour of each cell, which will account for small variations in shading and saturation. All the values described are used in the following major steps: A) Image processing (selecting a framing window and cropping everything outside of that window using an OpenCV method, B) Colour analysis (using the Huo et al. Robust Auto White-Balance API to account from distortions due to ambient lighting, creating separate image segments for each well in the biosensor using the OpenCV GrabCut algorithm, then inserting an image segment of each disk into a 2D array  to ensure that only the coloured wells are analyzed, after which the user will be prompted to mark a border around a disk with an on-screen drawing tool), and C) Approximating the relative expression of the reporter gene (we will analyze each cell in the array and the ratio of purple to yellow in the substrate, concentrations will be stored in another 2D array of the same size and can be used to indicate relative expression of the reporter gene based on the amount of purple pigment, which corresponds to the amount of chlorophenol-red-beta-D-galactopyranoside cleavage.)</p>
<p>Overall, we have created an app that will make colourimetric analysis simple and efficient for a layperson, and will be an invaluable tool for on-the-go testing when used in combination with paper-based biosensors (Pardee et al. 2014).</p>
</div>

<div>
<h5>Modelling Protein Folding with Rosetta</h5>
<p>We used Rosetta and pyRosetta to model and compare the gold-binding ability of GolS as a monomer and a predicted GolS homodimer. GolS belongs to the MerR family of transcriptional regulators, which usually function as homodimers. Based on the amino acid sequence of GolS, we generated a predicted 3D structure for a GolS monomer within Rosetta, and then docked the two GolS monomers together to create a homodimer. Following this, we modelled the ability of the predicted GolS homodimer to bind Gold(III), then compare the gold-binding abilities of two mutant versions of GolS created by our wetlab team.</p>
</div>


</html>
